csv_schema/models.py

```diff
@@ -33,24 +25,23 @@ def turn_preview_mode_off(sender, user, request, **kwargs):
 
 class DatabaseQueryset(BaseQuerySet):
     def viewable(self, user):
-        """ returns all tables that have columns
-        """
-        return self.filter(
-            table__in=Table.objects.viewable(user)
-        ).distinct().order_by(Lower('display_name'))
+        """Returns all Databases with Schemas that contain viewable Tables."""
+        schemas = Schema.objects.filter(tables__in=Table.objects.viewable(user))
+        return (self.filter(schemas__in=schemas)
+                    .distinct()
+                    .order_by(Lower('display_name')))
 
 
 class Database(BaseModel):
     SEARCH_FIELDS = [
         "display_name", "name", "description", "link"
     ]
     name = models.CharField(max_length=255, unique=True)
-    display_name = models.CharField(
-        max_length=255, unique=True, blank=True, null=True
-    )
+    display_name = models.TextField(blank=True, null=True)
```

Comment:
Why are we removing the uniqueness of db display name? Surely this is something we want?

Reply:
Lines [43](https://github.com/nhsengland/NCDR-reference-library/pull/51/files#diff-1a7ed64766867c4252cb60307a8d2d1aR43) and [65](https://github.com/nhsengland/NCDR-reference-library/pull/51/files#diff-1a7ed64766867c4252cb60307a8d2d1aR65) have the same `Name` value for different database names (`NHSE_Mental_Health` and `NHSE_MH_V3_YTD` respectively).

Do you think this is something we should push back on?